Artifacts/windows-azuredevops-multiple-agents/artifactfile.json

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+    "title": "Azure Pipelines Agent(s)",
+    "publisher": "Utkarsh Shigihalli",
+    "description": "Downloads latest Azure Pipelines agent for the account, and installs the specified number instances of the agents inside one VM. Registers the agent with the Azure DevOps account, and adds agents to the specified agent pool. Note - If the specified agent folder already exists it will be overwritten.",
+    "tags": [
+      "Azure DevOps",
+      "Pipelines",
+      "Release",
+      "Build",
+      "CI",
+      "Windows"
+    ],
+    "iconUri": "https://cdn.vsassets.io/content/icons/favicon.ico",
+    "targetOsType": "Windows",
+    "parameters": {
+      "Account": {
+        "type": "string",
+        "displayName": "Azure DevOps Account Name",
+        "description": "The name of the Azure DevOps account to add the build agent to. This is the value in your Azure DevOps URL: e.g. 'myaccount' in https://dev.azure.com/myaccount."
+      },
+      "PersonalAccessToken": {
+        "type": "securestring",
+        "displayName": "Azure DevOps Personal Access Token",
+        "description": "A personal access token with permissions to add build agents. It will only be used to register the agent."
+      },
+      "AgentName": {
```

Comment:
Are agent names with spaces allowed? Can you confirm that? Maybe there's some additional logic you may need to add, if that's not the case.

Reply:
You can create agents with space. Let me know if there is any change required.